.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: clojure
-lein: 2.7.1
+lein: 2.8.1
 dist: trusty
 jdk:
   - oraclejdk8
+  - oraclejdk10
```

Comment:
Do we want to test on both version or just 8/10 (and if so, which one)?

Reply:
@austb why not  - oraclejdk9 ?  Would that be a different ticket?  Was this the thing you mentioned about tags?  Ticket commit message mentions Java 9.

Reply:
9 and 10 don't seem to be very different, 10 was sort of a "trial run" for a faster release cadence for the JDK. Also the end goal is Java 11 support, so I didn't think it was worth testing.

Reply:
@pcarlisle I noticed you had 7, 8, and 9 here:  https://github.com/puppetlabs/clj-kitchensink/pull/99/files
Do you have any preference?

Reply:
i think 8 and 10 is fine